source/snippets/troubleshooting.txt

```diff
@@ -0,0 +1,128 @@
+.. _snip-troubleshooting:
+
+===============
+Troubleshooting
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. warning:: Experimental feature
+
+   .. include:: /includes/warnings/warn-experimental-feature.rst
+
+The following sections provide troubleshooting suggestions.
+
+.. _snip-npm-logs:
+
+View ``npm`` Log Files
+----------------------
+
+The ``npm`` log files are a good place to start if you encounter a
+problem. The log file location will vary depending on your ``npm``
+installation. It will be something like:
+
+.. code-block:: shell
+
+   /<NPM USER HOME>/.npm/_logs/2021-09-16T22_03_34_534Z-debug.log
+
+When you locate the log files, check the most recent one.
+
+Non-specific Error Message
+--------------------------
+
+**Problem**: :binary:`~bin.mongosh` returns a non-specific error
+message when you try to start the shell.  
```

Comment:
I assume this is something that you ran into at some point – do you remember details? The right solution here is to actually provide a specific error message, of course :)

Reply:
It was, I'm not sure what though. Most likely it happened when I was testing out config settings. I'll let you know if I run into it again.